lib/control/v1/cubbyhole.js

```diff
@@ -7,11 +7,9 @@ function CubbyHole(storage) {
     const token = req.params.token;
     const path = req.params.path;
 
-    storage.lookup(token, path, CubbyHoleProvider, (err, result) => {
-      const response = (err) ? {error: err.message} : result;
-
-      res.json(response);
-    });
+    storage.lookup(token, path, CubbyHoleProvider)
+      .then((result) => res.json(result))
+      .catch((err) => res.json({error: err.message}));
```

Comment:
Errors should get passed to the controller's next handle so that they can be handled consistently by another layer